packages/overlay/src/loader.ts

```diff
@@ -10,6 +10,33 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+import type {
```

Comment:
Do you mind writing some comments on this file just to help everyone understand what we are doing here?

Reply:
Added.